### PR TITLE
[RA1 Ch01] Terminology section completed

### PR DESCRIPTION
**Pull request by karinesevilla**
Fixes issue #3106

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 1]
```diff
@@ -146,10 +146,233 @@ to address them (for example, resources autoscaling).
...
+controlling and managing the Network Function Virtualisation
+Infrastructure (NFVI) compute, storage and network resources.
+
+**Virtual Machine (VM):** virtualised computation environment that
+behaves like a physical computer/server.
+A **VM** consists of all of the components (processor (CPU),
+memory, storage, interfaces/ports, etc.) of a physical computer/server.
+It is created using sizing information or Compute Flavour.
+
+**Virtual Network Function (VNF):** A software implementation of a
```
Comment (pgoyal01) [inline doc/ref_arch/openstack/chapters/chapter01.rst:356]:
```suggestion
**Virtualised Network Function (VNF):** A software implementation of a
```

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 2]
```diff
@@ -146,10 +146,233 @@ to address them (for example, resources autoscaling).
...
+(APIs) and the SDS controller software provides storage access services
+and networking.
+
+**Tenant:** cloud service users sharing access to a set of physical and
+virtual resources, ITU-T Y.3500.
+Tenants represent an independently manageable logical pool of
+compute, storage and network resources abstracted from physical
+hardware.
+
+**Tenant Instance:** refers to a single **Tenant**.
```
[the remark this answers is not in the document]
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: @pgoyal01 Suggestion committed

File: doc/ref_arch/openstack/chapters/chapter01.rst (hunk 1 above)
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: @pgoyal01 Suggestion committed

**Comment by karinesevilla** [comment]
Decision taken the 29th of August to include more Terminology per Pankaj comments + use Glossary key word

**Review by walterkozlowski — APPROVED** [review]

**Comment by karinesevilla** [comment]
@collivier @pgoyal01 I have added new terms used in Chapter Conformance. Please review.

**Review by pgoyal01 — APPROVED** [review]

**Review by collivier — APPROVED** [review]

**Pull request merged**